src/cryptoadvance/specter/api/rest/resource_jwt.py

```diff
@@ -0,0 +1,102 @@
+import jwt
+from flask import current_app as app
+from cryptoadvance.specter.api.rest.base import (
+    BaseResource,
+    rest_resource,
+    AdminResource,
+)
+import uuid
+import datetime
+import logging
+from ...user import *
+from .base import *
+
+from .. import auth
+
+logger = logging.getLogger(__name__)
+
+def generate_jwt(user):
+    payload = {
+        "user": user.username,
+        "exp": datetime.datetime.utcnow() + datetime.timedelta(days=1),
+    }
+    return jwt.encode(payload, app.config["SECRET_KEY"], algorithm="HS256")
+
+def generate_token_id():
+    return str(uuid.uuid4())
+
+@rest_resource
+class ResourceJWT(SecureResource):
+    ''' 
+      A Resource to manage JWT tokens in order to authenticate against the REST-API
+      Other then the other Resources, this endpoint uses BasicAuth to avoid the chicken egg problem
+      This one is only to create a token. The other Resource for getting and deleting.
+      This violates the REST principles but only on the implementation-side. Happy for improvements here.
+    '''
+    endpoints = ["/v1alpha/token/"]
+
+    def post(self):
```

Comment:
this is suppose to be a get-request, no?! Returning `jwt_tokens` directly also expose the secret, no?!

Reply:
Done via nested hashmap